src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx

```diff
@@ -177,11 +181,10 @@ export default function FormStartCompute({
   useEffect(() => {
     if (!asset?.accessDetails || !selectedAlgorithmAsset?.accessDetails?.length)
       return
-
-    setDatasetOrderPrice(datasetOrderPriceAndFees?.price || accessDetails.price)
+    // setDatasetOrderPrice(datasetOrderPriceAndFees?.price || accessDetails.price)
     setAlgoOrderPrice(
       algoOrderPriceAndFees?.price ||
-        selectedAlgorithmAsset?.accessDetails?.[0]?.price
+        selectedAlgorithmAsset?.stats?.price?.value.toString()
```

Comment:
stats field will be deprecated since we have multiple services. We don't have UI for picking services when picking algorithms. So for now I would just use [0] service. Maybe you can also add a TODO comment